src/libraries/System.Net.Http/src/Resources/Strings.resx

```diff
@@ -264,6 +264,9 @@
   <data name="net_http_value_must_be_greater_than" xml:space="preserve">
     <value>The specified value must be greater than {0}.</value>
   </data>
+  <data name="net_http_value_must_be_greater_than_or_equal" xml:space="preserve">
+    <value>The specified value must be greater than or equal {0}.</value>
```

Comment:
```suggestion
    <value>The specified value must be greater than or equal to {0}.</value>
```
would it be useful to include the "specified value" ? Sometimes exception messages are all you have out of a log.

Reply:
Fixed.